lib/Rx/Observable.php

```diff
@@ -839,4 +840,19 @@ public function timeout($timeout, ObservableInterface $timeoutObservable = null,
             return new TimeoutOperator($timeout, $timeoutObservable, $scheduler);
         });
     }
+
+    /**
+     * Projects each element of an observable sequence into zero or more buffers which are produced based on
+     * element count information.
+     *
+     * @param $count
+     * @param int $skip
+     * @return AnonymousObservable
+     */
+    public function bufferWithCount($count, $skip = -1)
```

Comment:
Is the `$skip` useful for you? If not I'd propose shipping without it first. It seems to make things more complicated? At least in terms of understanding what the operator does.

If you want to keep it that's fine as well, but can we make `= null` the value for if you don't want to skip?